cmd/utils/manifest.go

```diff
@@ -1,33 +0,0 @@
-// Copyright 2022 The Okteto Authors
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package utils
-
-import (
-	"path/filepath"
-
-	oktetoLog "github.com/okteto/okteto/pkg/log"
-	"github.com/okteto/okteto/pkg/model"
-)
-
-// InferName infers the application name from the folder received as parameter
-func InferName(cwd string) string {
```

Comment:
Moved to a new package in this commit https://github.com/okteto/okteto/pull/3300/commits/d95d8c19d042cd65e120171410dec5f162275fd9